### PR TITLE
minor documentation fix

### PR DESCRIPTION
**Pull request by Jagadish-NXP**
#### Problem
 1. Documentation update for k32w specific elock example
 2. k32 SDK patching script had an error on macos

 #### Summary of Changes
 1. Fixed documentation to properly identify buttons with the expected behaviour
2. Make the k32w SDK patching script compatible to both linux and macos

### CODE REVIEW

File: third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [[ ! -v K32W061_SDK_ROOT ]]; then
```
Comment (andy31415) [inline third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh:3]: This seems to not be a documentation change - should we call it out in the description?

Comment (Jagadish-NXP) [reply third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh:3]: It is a check to make it compatible for linux as well as macos. The line checks if the k32w sdk exists before applying the patch. This has been verified to work.

**Review by mspang — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

File: third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh (hunk 1 above)
Comment (bzbarsky-apple) [reply third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh:3]: No, I don't get it.  Shouldn't this be `-d $K32W061_SDK_ROOT`?  

What, exactly, has been verified to work?  I just tested this locally, and I get:
```
% mkdir /tmp/foo                                      
% env K32W061_SDK_ROOT=/tmp/foo third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh
K32W061_SDK_ROOT is not set
```

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh (hunk 1 above)
Comment (Jagadish-NXP) [reply third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh:3]: If you refer to https://github.com/project-chip/connectedhomeip/blob/master/examples/lock-app/k32w/README.md
the document explains how to download the SDK, Once the SDK is downloaded, "export K32W061_SDK_ROOT=path_to_SDK" has to be executed as explained in the documentation.

Comment (bzbarsky-apple) [reply third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh:3]: I understand all that, but that does not affect whether this specific line works, and this specific line does not work.  Can you please paste in a shell session showing you running these comands against the modified script?

Comment (Jagadish-NXP) [reply third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh:3]: Ok I get it.. Thanks!! have made an update

**Review by bzbarsky-apple — APPROVED** [review]
Thank, now I buy this.

**Comment by bzbarsky-apple** [comment]
@saurabhst @BroderickCarlin @jelderton @hawk248 could you take a look, please?

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@saurabhst @jelderton ?

**Comment by woody-apple** [comment]
@saurabhst ?

**Review by saurabhst — APPROVED** [review]

**Pull request merged**